src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj

```diff
@@ -1,4 +1,4 @@
-<Project>
+﻿<Project>
```

Comment:
Undo changes from this file.  We generally avoid whitespace-only changes to files, and updating the byte order mark is included.